src/handler.rs

```diff
@@ -89,6 +96,9 @@ where
     atomic_mem: GM<B>,
     vrings: Vec<V>,
     worker_threads: Vec<thread::JoinHandle<VringEpollResult<()>>>,
+    inflight_file: Option<File>,
+    inflight_mapping_addr: Option<GuestAddress>,
+    inflight_mmap_size: usize,
```

Comment:
It would better to introduce a `struct InflightFdState`, and move all related methods to this new object.